fdbclient/SnapshotCache.h

```diff
@@ -245,6 +245,7 @@ class SnapshotCache {
 		}
 
 		bool operator == ( const iterator& r ) const { return it == r.it && offset == r.offset; }
+		bool operator!=(const iterator& r) const { return !(*this == r); }
```

Comment:
nit:  Inconsistent whitespace between existing `operator == ` and  new `operator!=` throughout the PR.  Will `git clang-format` find + fix them all?

Anyway, mixing whitespace changes into this PR won't make it easier to proofread, so I wouldn't bother fixing it now.

Reply:
I ran `git clang-format`. It didn't fix the whitespace for operator== since I didn't touch that line